cypress/support/step_definitions/dashboard/i_am_logged_in_as_a_string.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable */
```

Comment:
The changes I mentioned in [an earlier comment](https://github.com/UKForeignOffice/lists/pull/452/files#r952851070) needed to be in place before I could disable eslint from checking these sorts of files.